app/views/components/cash_account_balance.scala.html

```diff
@@ -45,12 +46,12 @@
             classes = "govuk-label govuk-label--l govuk-!-margin-bottom-7 govuk-!-margin-top-2",
             id = Some("balance-available")
         )
-        @if(showLastTransactionsHeading){
-            @h2(
-                msg = "cf.cash-account.detail.last-transactions",
-                classes="govuk-heading-m",
-                id = Some("last-transactions")
-            )
-        }
     }
 
+    @if(showBalance && displayLastSixMonthsHeading) {
```

Comment:
Why do we need to pass another boolean ?

Reply:
we are not passing another boolean, we are making use of existing boolean condition. It is done, as earlier it was a nested if (if inside if) and not readable especially for the view. Hence this change has been done and its just refactoring.